src/lib.rs

```diff
@@ -105,3 +104,33 @@ pub fn Icon(
     svg = svg.inner_html(icon.data);
     leptos::IntoView::into_view(svg)
 }
+
+macro_rules! impl_from {
+    ($feature:expr, $:icon_type) => {
```

Comment:
Here there is a mistake on `$:icon_type`, this should probably be `$icon_type:ident`.